### DISCUSSION TITLE
Latest Blast /SampleAssetViewer_x64 with scenes?

### DISCUSSION THREAD

**Discussion by scomnoob** [NVIDIA-Omniverse/PhysX#266] [General]
Hello

How can i get  latest Blast /SampleAssetViewer_x64.exe with test scenes (Castle...etc)  ?

Only i can found - is old version https://github.com/NVIDIAGameWorks/Blast , but scripts are outdated in it (dead links)

Many thanks

**Comment by nvidia-bgaldrikian** [comment]
Hello,

I'm sorry to say we aren't supporting SampleAssetViewer in the new public repository.  It may revived at some future time, but currently it's not scheduled.

Bryan Galdrikian
